cpp/modmesh/view/RManager.cpp

```diff
@@ -26,12 +26,11 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <modmesh/view/RMainWindow.hpp> // Must be the first include.
+#include <modmesh/view/RManager.hpp> // Must be the first include.
```

Comment:
Code in `RApplication` and `RMainWindow` is merged here.  The diff is messy.  It's probably too hard for review.